src/mlpack/tests/random_forest_test.cpp

```diff
@@ -324,19 +321,19 @@ BOOST_AUTO_TEST_CASE(LeafSizeDatasetTest)
   size_t majorityClass = predictions[0];
   arma::vec majorityProbs = probabilities.col(0);
 
-  BOOST_REQUIRE_EQUAL(probabilities.n_rows, 3);
-  BOOST_REQUIRE_EQUAL(probabilities.n_cols, dataset.n_cols);
-  BOOST_REQUIRE_EQUAL(predictions.n_elem, dataset.n_cols);
+  REQUIRE(probabilities.n_rows == 3);
+  REQUIRE(probabilities.n_cols == dataset.n_cols);
+  REQUIRE(predictions.n_elem == dataset.n_cols);
   for (size_t i = 1; i < predictions.n_cols; ++i)
   {
-    BOOST_REQUIRE_EQUAL(predictions[i], majorityClass);
+    REQUIRE(predictions[i] == majorityClass);
     for (size_t j = 0; j < probabilities.n_rows; ++j)
-      BOOST_REQUIRE_CLOSE(probabilities(j, i), majorityProbs[j], 1e-5);
+      REQUIRE(probabilities(j, i) == Approx(majorityProbs[j]).epsilon(1e-7));
   }
 }
 
 // Make sure we can serialize a random forest.
-BOOST_AUTO_TEST_CASE(SerializationTest)
+TEST_CASE("RandomForestSerializationTest", "[RandomForestTest]")
```

Comment:
Here, I slightly changed the name of the test case. When I retained the name 'SerializationTest', I got the following error 

`error: TEST_CASE( "SerializationTest" ) already defined. First seen at mlpack/mlpack/src/mlpack/tests/random_forest_test.cpp:337 Redefined at mlpack/mlpack/src/mlpack/tests/sparse_coding_test.cpp:140`

So I changed the name to 'RandomForestSerializationTest'. Kindly let me know if this is ok.

Reply:
Yes, that's completely fine.